src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java

```diff
@@ -585,9 +585,10 @@ public void verifySpecifiedPanelIdsAreCollapsed(String[] ids) {
     }
 
     public boolean isSectionPanelExist(String section) {
-        List<WebElement> panels = browser.driver.findElements(By.cssSelector("div[id^='panelHeading-']"));
```

Comment:
Is this change relevant to his PR?

Reply:
Yup. This function is buggy. It can only verify if its true. if its false it throws an exception. I fixed it to use assertFalse with it.